packages/fxa-auth-server/test/test_server.js

```diff
@@ -124,3 +116,26 @@ TestServer.prototype.uniqueUnicodeEmail = function() {
 };
 
 module.exports = TestServer;
+
+function patchVerify(server) {
+  // delete the existing /v1/verify route
+  // unfortunately hapi doesn't have a public api for it
+  const xs = server._core.router.routes.post.routes;
+  xs.splice(xs.findIndex(x => x.path === '/v1/verify'), 1);
+  delete server._core.router.routes.post.router._fulls['/v1/verify'];
+
+  try {
+    server.route([
+      {
+        method: 'POST',
+        path: '/v1/verify',
+        handler: async function(request, h) {
+          const data = JSON.parse(Buffer.from(request.payload.token, 'hex'));
+          return h.response(data).code(data.code || 200);
+        },
+      },
+    ]);
+  } catch (e) {
+    console.error(e);
+  }
+}
```

Comment:
woof. this was gnarly. this replaces what `oauth_helper.js` did. can't say I agree with it but I'll be damned if I'm gonna rewrite all the api tests to change it.

Reply:
Can you leave a note about why you patch the verify route to avoid that context being lost?